src/components/NcDialog/NcDialog.vue

```diff
@@ -114,8 +115,14 @@ export default {
 		return {
 			showDialog: false,
 			newName: '',
-			currentName: 'none yet.',
+			currentName: 'non yet.',
 			buttons: [
+				{
+					label: 'Reset',
+					nativeType: 'reset',
+					// returning `false` to prevent the dialog from closing
+					callback: () => false,
```

Comment:
Should it be the default behavior for reset?

Reply:
this makes sense!